src/test/resources/parser_tests/inputs/testcase20.txt

```diff
@@ -0,0 +1,8 @@
+//> using scala 2.13.15
```

Comment:
Do we really use this tastcase anywhere?

Reply:
Yes, it is a duplicate of 
 `String numericScalaVersionDirective = "//> using scalaFullNumeric 2.4.15 “;`

Reply:
@pikinier20